### PR TITLE
Update New features and enhancements

### PR DESCRIPTION
**Pull request by jowens-axway**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by jowens-axway** [comment]
@dginevaAxway please review also. Text is very similar to the tech enablement slides.
Apologies I cannot remember how to find the deploy preview link to add it above.

**Comment by dginevaAxway** [comment]
https://deploy-preview-1700--axway-open-docs.netlify.app/docs/apim_relnotes/20210330_apip_relnotes/
on jira item IAP-3988 I can't edit it

**Comment by dginevaAxway** [comment]
looks good for me

File: content/en/docs/apim_relnotes/20210330_apip_relnotes.md [hunk 1]
```diff
@@ -19,23 +19,26 @@ API Portal is available as a software installation or a virtualized deployment i
 
 ## New features and enhancements
 
-### placeholder 1
+### Sample Helm chart for Axway Managed Cloud (Internal release only)
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20210330_apip_relnotes.md:22]: What's this sample helm chart? Why/ How to use it?
Is it documented somewhere? (For more information, see...)
Is this PR related to a Jira ticket?

File: content/en/docs/apim_relnotes/20210330_apip_relnotes.md [hunk 2]
```diff
@@ -19,23 +19,26 @@ API Portal is available as a software installation or a virtualized deployment i
...
+* The sample helm chart, once configured for the Axway Managed Cloud Azure environment will allow easier installation of API Portal in this environment.
+* It is currently intended for use by the Axway Managed Cloud team in an Azure environment only.
+* In a future update, this sample will be genericized and made available to external customers.
 
-For more information, see:
+### Unattended mode enhanced to use a configuration file for parameters
 
-* [some reference 1](/docs/placeholder)
-* [some reference 2](/docs/placeholder)
+* Unattended mode uses a total of 24 parameters, all of which must be specified on the command line.
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20210330_apip_relnotes.md:30]: Is the documentation updated for this? Can you add a link 'For more info, see..'?

**Review by dginevaAxway — APPROVED** [review]

File: content/en/docs/apim_relnotes/20210330_apip_relnotes.md (hunk 2 above)
Comment (jowens-axway) [reply content/en/docs/apim_relnotes/20210330_apip_relnotes.md:30]: added the link there although it is to the develop branch - so will this link need to be updated again once live to point to the published page? @andreamussap 
https://developmar21--axway-open-docs.netlify.app/docs/apim_installation/apiportal_install/install_unattended/

File: content/en/docs/apim_relnotes/20210330_apip_relnotes.md (hunk 1 above)
Comment (jowens-axway) [reply content/en/docs/apim_relnotes/20210330_apip_relnotes.md]: on reflection, this helm chart is an internal delivery to the managed cloud team so am removing this section from the release notes for now

File: content/en/docs/apim_relnotes/20210330_apip_relnotes.md (hunk 2 above)
Comment (andreamussap) [reply content/en/docs/apim_relnotes/20210330_apip_relnotes.md:24]: The links always start at **/docs**, so we don't need to worry about the branch

**Review by andreamussap — APPROVED** [review]
